source/client-side-operations-timeout/etc/templates/global-timeoutMS.yml.template

```diff
@@ -57,11 +57,11 @@ tests:
           client: *failPointClient
           failPoint:
             configureFailPoint: failCommand
-            mode: { times: 1 }
+            mode: { times: 2 }
```

Comment:
Why are we changing `times` for a lot of these fail points? Does it have to do with modifications to retryability you've made in Python? I haven't changed our retry logic yet in the Go driver, but the spec currently says to retry infinitely in the presence of `timeoutMS`; looks like maybe you've changed Python to retry twice.

Reply:
This was actually required for a different reason. In Python on Windows it's actually possible for the socket read (technically socket poll()) to raise a timeout 20ms _before_ the actual deadline. So a timeout gets raised but then the retry logic sees that there is 20ms remaining so a retry is attempted. Without this change the retry proceeds quickly and the operation actually succeeds within the timeout.

Reply:
Hmm ok. That does seem Python-specific; do you think other languages will have issues with socket-level early timeouts followed by a successful retry? I'm fine increasing `times` as it doesn't affect drivers that fail on the first hit to the fail point, but maybe we could leave a comment in the template explaining why it's necessary?

Reply:
Done.